pkg/installation/installation.go

```diff
@@ -39,9 +40,10 @@ type ComponentsConfig struct {
 
 // Installation contains the installation elements and configuration options.
 type Installation struct {
+	Docker      docker.KymaDockerService
 	K8s         kube.KymaKube
 	Service     Service
-	currentStep step.Step
+	CurrentStep step.Step
```

Comment:
Why is this field now exported?

Reply:
its used in the Docker package with in pushKymaInstaller method so needed to be exported

Reply:
You are now passing it as a parameter to `PushKymaInstaller ` so it does not need to be exported

Reply:
yeah fixed this.